jdsol/1.0/_index.md

```diff
@@ -0,0 +1,53 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Debugging Support for Other Languages Spec 1.0"
+date: 2019-07-18T16:10:38-04:00
+summary: "First Version Spec for DSOL"
+---
+Jakarta Debugging Support for Other Languages provides a mechanism by which 
+programs executed under the JavaTM virtual machine but written in languages 
+other than the Java programming language, can be debugged with references to i
+the original source (for example, source file and line number references).
+
+* [Jakarta Debugging Support for Other Languages 1.0 Specification Document](./jdsol_1.0.pdf) (PDF)
```

Comment:
Where are these generated from?  What github repo?  I found the JDSOL TCK repo, but not one for the spec and api...

Reply:
generated from https://github.com/eclipse-ee4j/jsp-api/tree/master/jdsol-spec, the artifacts are copied here